example/simple_run_through/index.dart

```diff
@@ -1,31 +1,30 @@
 library index;
 
 import 'package:lawndart/lawndart.dart';
-import 'dart:html';
-import 'dart:indexed_db';
+import 'package:web/web.dart';
 
-runThrough(Store store, String id) async {
-  var elem = querySelector('#$id')!;
+void runThrough(Store store, String id) async {
+  var elem = document.getElementById('$id')!;
 
   try {
     await store.nuke();
     await store.save(id, "hello");
     await store.save("is fun", "dart");
     await for (var value in store.all()) {
-      elem.appendText('$value, ');
+      elem.innerHTML += '$value, ';
```

Comment:
I get:
`The operator '+' isn't defined for the type 'JSAny'.
Try defining the operator '+'.`

Reply:
Ah, I must've missed this when Dart switched `innerHTML` to `JSAny` instead of `String`.